src/Streamly/Internal/FileSystem/File.hs

```diff
@@ -222,6 +232,12 @@ readChunksWith :: (MonadCatch m, MonadAsync m)
 readChunksWith size file =
     withFile file ReadMode (FH.readChunksWith size)
 
+{-# DEPRECATED toChunksWithBufferOf "Please use 'readChunksWith' instead" #-}
```

Comment:
You've added and deprecated these?

Reply:
Were these removed previously?

Reply:
Yes. were renamed without deprecating the older ones.